login.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Login Page</title>
+  <link href="login.css" rel="stylesheet" />
+</head>
+
+<body>
+  <form action="/my-handling-form-page" method="post">
+    <ul>
+      <li>
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="user_name" />
```

Comment:
the input should be prompt the user to enter their password not name
Also email input should be listed first